uberon_edit.obo

```diff
@@ -193949,6 +193949,39 @@ intersection_of: in_left_side_of UBERON:0010709 ! pelvic complex
 property_value: implements_design_pattern https://github.com/obophenotype/uberon/wiki/Modeling-paired-structures-Design-Pattern xsd:anyURI
 created_by: http://orcid.org/0000-0001-9897-3238
 
+[Term]
+id: UBERON:8400001
+name: hepatic acinus periportal zone
+def: "The region of the hepatic acinus closest to the portal triad. Blood in this zone has the highest oxygen concentration of the three metabolic zones in the hepatic acinus." [PMID:28126520]
+synonym: "hepatic acinus zone I" EXACT []
+intersection_of: adjacent_to UBERON:0001279 ! portal triad
```

Comment:
Needs a superclass.   I think 'organ subunit' http://purl.obolibrary.org/obo/UBERON_0000063 would work.  Not sure what the equivalentClass definition buys us and whether it will be safe (someone might define some sub-region that fits this description and you probably wouldn't want it auto-classified.  So I'd switch this all to SubClassOf